Editor/UI/Windows/CourseCreationWizard.cs

```diff
@@ -14,7 +14,7 @@ internal class CourseCreationWizard : EditorWindow
         private static CourseCreationWizard window;
 
         // CourseCreationWizard is obsolete and was replaced by CreatorSetupWizard
-        // private const string menuPath = "Innoactive/Creator/Create New Course...";
+        // private const string menuPath = "Innoactive/Create New Course...";
```

Comment:
```suggestion
```

Reply:
Do you know why this was here in the first place?